framework_tvb/tvb/interfaces/web/static/js/bursts.js

```diff
@@ -661,9 +665,21 @@ function launchNewBurst(currentForm, launchMode) {
     });
 }
 
+function setInitialFocusOnButton(simulator_params){
+    const current_url = simulator_params.lastElementChild.action;
+    if(current_url !== undefined && current_url.includes('setup_pse')){
+        document.getElementById('launch_simulation').focus();
+    }else if(current_url !== undefined && current_url.includes('launch_pse')){
+        document.getElementById('launch_pse').focus();
+    }else {
+        const next_buttons = document.getElementsByClassName('btn btn-primary next');
```

Comment:
I suggest here we make a specific class, easy to search at a possible future rename.
e.g. 'next-button'

Reply:
It is preferably to use JQuery `$('#launch_simulation').focus()` instead of `document.getElementById('launch_simulation').focus()` as usually is faster and more realiable